grails-app/utils/org/broadinstitute/orsp/utils/UtilityClass.groovy

```diff
@@ -37,4 +41,26 @@ class UtilityClass {
             return output
         }
     }
+
+    /**
+     * Register Issue's object JSON mapping for Review Category Report
+     */
+    static void registerIssueMarshaller() {
+        JSON.createNamedConfig('issue') {
```

Comment:
I haven't seen this before, but it looks nice. It would be nicer if the name were a static that were a little more indicative of what this does, something like 
```
public static final String ISSUE_RENDERER_CONFIG = 'issue'
...
JSON.createNamedConfig(ISSUE_RENDERER_CONFIG)
```